pol-core/ecompile/ECompileMain.cpp

```diff
@@ -466,7 +466,7 @@ int readargs( int argc, char** argv )
         vlev = atoi( &argv[i][2] );
         if ( !vlev )
           vlev = 1;
-        Legacy::Compiler::setVerbosityLevel( vlev );
+        compilercfg.VerbosityLevel = vlev;
```

Comment:
The "-v" flag also enables the variable "verbose" which is used in compile_file(). Did you leave that one out on purpose?

Reply:
No, I missed that.  I will take another look.  Thanks!

Reply:
I removed the `verbose` flag and instead check `compilercfg.VerbosityLevel`.